### ISSUE TITLE
Windows detection bug in Get-FrequentFolders

### ISSUE THREAD

**Issue by nheath** [vors/ZLocation#100]
Importing the module on osx powershell core prints the following error output:

```
New-Object : A parameter cannot be found that matches parameter name 'ComObject'.
At /Users/nheath/.local/share/powershell/Modules/ZLocation/1.4.0/ZLocation.psm1:247 char:39
+             $QuickAccess = New-Object -ComObject shell.application
+                                       ~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (:) [New-Object], ParameterBindingException
+ FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.NewObjectCommand

You cannot call a method on a null-valued expression.
At /Users/nheath/.local/share/powershell/Modules/ZLocation/1.4.0/ZLocation.psm1:248 char:13
+             $QuickAccess.Namespace("shell:::{679f85cb-0220-4080-b29b- ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : InvokeMethodOnNull
```

In the following code, `Get-Variable` is returning `System.Management.Automation.PSVariable` which implicitly cast to `[bool]` is always `$true`. Instead, use its `Value` property
https://github.com/vors/ZLocation/blob/87fe00bc82103cda3956f212a9a08511a2579596/ZLocation/ZLocation.psm1#L244

**Comment by theaquamarine** [comment]
Thanks very much! I've incorporated your fix, added macOS to our CI, and [released a new version, v1.4.1](https://www.powershellgallery.com/packages/ZLocation/1.4.1) with the fix included.

**Comment by nheath** [comment]
👍 thanks!
